sw_watchdog/launch/windowed_watchdog_lifecycle.launch.py

```diff
@@ -1,49 +1,42 @@
 # Copyright (c) 2020 Mapless AI, Inc.
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
+# Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
Since this license text is copied from the license, the quotes should be left alone.

Reply:
the double quote is considered a 'bad quote' by flake. I will check that I ignore that warning and change it back to double quotes.

Reply:
No, this was just a string replace mistake. Good catch!